PurchasesCoreSwift/Public/Purchases.swift

```diff
@@ -0,0 +1,1275 @@
+     */
+    @objc static var logLevel: LogLevel {
+        get { Logger.logLevel }
+        set { Logger.logLevel = newValue }
+    }
+
+    /**
+     * Set this property to your proxy URL before configuring Purchases *only* if you've received a proxy key value
+     * from your RevenueCat contact.
+     */
+    @objc static var proxyURL: URL? {
+        get { SystemInfo.proxyURL }
+        set { SystemInfo.proxyURL = newValue }
+    }
+
+    /**
+     * Set this property to true *only* if you're transitioning an existing Mac app from the Legacy
+     * Mac App Store into the Universal Store, and you've configured your RevenueCat app accordingly.
+     * Contact support before using this.
+     */
+    @objc static var forceUniversalAppStore: Bool {
+        get { SystemInfo.forceUniversalAppStore }
+        set { SystemInfo.forceUniversalAppStore = newValue }
+    }
+
+    /**
+     * Set this property to true *only* when testing the ask-to-buy / SCA purchases flow. More information:
+     * http://errors.rev.cat/ask-to-buy
+     */
+    @available(iOS 8.0, macOS 10.14, watchOS 6.2, macCatalyst 13.0, *)
+    @objc static var simulatesAskToBuyInSandbox: Bool {
+        get { StoreKitWrapper.simulatesAskToBuyInSandbox }
+        set { StoreKitWrapper.simulatesAskToBuyInSandbox = newValue }
+    }
+
+    /**
+     * Enable debug logging. Useful for debugging issues with the lovely team @RevenueCat.
+     */
+    @objc static var debugLogsEnabled: Bool {
+        get { logLevel == .debug }
+        set { logLevel = newValue ? .debug : .info }
+    }
+
+    /**
+     * Set a custom log handler for redirecting logs to your own logging system.
+     * By default, this sends Info, Warn, and Error messages. If you wish to receive Debug level messages,
+     * you must enable debug logs.
+     */
+    @objc static var logHandler: (LogLevel, String) -> Void {
+        get { Logger.logHandler }
+        set { Logger.logHandler = newValue }
+    }
+
+    /// Current version of the Purchases framework.
+    @objc static var frameworkVersion: String { SystemInfo.frameworkVersion }
+
+    @objc var allowSharingAppStoreAccount: Bool {
+        get { purchasesOrchestrator.allowSharingAppStoreAccount }
+        set { purchasesOrchestrator.allowSharingAppStoreAccount = newValue }
+    }
+
+    @objc var finishTransactions: Bool {
+        get { systemInfo.finishTransactions }
+        set { systemInfo.finishTransactions = newValue }
+    }
+
+    @objc static var sharedPurchases: Purchases {
+        if let purchases = purchases {
+            return purchases
+        }
+
+        notConfiguredAssertionFunction()
+
+        // This will only be returned during testing. #hack for ARM64 testing fatalError()
+        return PurchasesTestStandIn(apiKey: "StandIn", appUserID: nil)
+    }
+
+    static var shared: Purchases { sharedPurchases }
+    @objc static var isConfigured: Bool { purchases != nil }
+
+    private static var purchases: Purchases?
```

Comment:
nitpick: I think it'd make sense to also keep the delegate around these, since they're kind of related (in that you might use them together)

Reply:
Good point. Moving.